plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -165,7 +177,8 @@ export class PersonState {
     }
 
     private async updatePersonProperties(): Promise<Person> {
-        const personFound = await this.db.fetchPerson(this.teamId, this.distinctId)
+        // Note: We always fetch the person anew to avoid property-related races.
```

Comment:
I'm confused it says that we always fetch the person and we should (based on current property updates), but the code uses `this.person`, which potentially came from merging, which could be in a different kafka partition, i.e. processed in parallel to updates to this person.

Reply:
Comment is out of date, will remove.

Reply:
I think if we use person we get from merging there could be other updates which we would be overriding if we drop this, because of the way we do property updates, that we should solve that problem as a whole, so it's fine to make this a bit worse ... probably ... just be aware that this can cause some props updates to be lost.